src/main/java/technology/semi/weaviate/client/v1/misc/model/PqConfig.java

```diff
@@ -0,0 +1,30 @@
+package technology.semi.weaviate.client.v1.misc.model;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+
+@Getter
+@Builder
+@ToString
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+public class PqConfig {
```

Comment:
just a suggestion maybe it should be named `PQConfig`?

Reply:
renamed